xbmc/windowing/windows/WinSystemWin32.cpp

```diff
@@ -216,10 +216,11 @@ bool CWinSystemWin32::CreateNewWindow(const std::string& name, bool fullScreen,
   UpdateWindow( m_hWnd );
   
   // Configure the tray icon.
+  m_trayIcon.hBalloonIcon = NULL;
```

Comment:
This should either be in the ctor or in the header file

Reply:
meh, it's probably fine

Reply:
can this be `nullptr` ?